torchci/lib/bot/autoLabelBot.ts

```diff
@@ -1,10 +1,71 @@
 import { Context, Probot } from "probot";
 
-const regexToLabel: [RegExp, string][] = [
+const titleRegexToLabel: [RegExp, string][] = [
   [/rocm/gi, "module: rocm"],
   [/DISABLED\s+test.*\(.*\)/g, "skipped"],
 ];
 
+const filenameRegexToReleaseCategory: [RegExp, string][] = [
```

Comment:
Do we like the order with which these labels would appear? The OG script would break on the first match of the first file, so order matters.

Reply:
Hmm can a commit match with two of these patterns? If that is the case, one thing I'd do is order the categories that are more wishy-washy at the end, e.g. "python_frontend" or "cpp".

Reply:
A commit with two diff patterns changed would match multiple, but since we take the first, we just wouldn't get the second.

I'll move python_frontend and cpp last!